style.css

```diff
@@ -1,4 +1,74 @@
 body {
-background-color: #232323
+  color: #ffffff;
+  font-family: 'Ubuntu', sans-serif;
+  font-size: 15px;
+  width: auto;
+  background: url(nieckovsky_image_main.jpg);
+  background-color: #111415;
+  background-size: 100%;
+  background-position: center;
+  background-attachment: fixed;
+  background-repeat: no-repeat;
+}
+
+header {
+  width: 100%;
+  height: 50px;
+}
+
+nav {
+  display: inline-block;
+  width: 100%;
+  position: relative;
```

Comment:
I don't think position: relative is needed here. Same goes for margins.

Reply:
I've decided to give it position: relative, so the elements inside it will be absolutely positioned relative to it. It seemed like a good idea since absolute positioning needs at least a parent with a declared position other than static. Am I absolutely in the wrong here?
As for the margins, I can see how it's kinda sloppy; will be changed.

Reply:
You're not wrong here, in CSS there's always more than one way to solve a problem. Relative just looked hacky to me, but if you would like to leave it I won't protest. If it gets job done  that is.